source/MaterialXGenGlsl/GlslResourceBindingContext.cpp

```diff
@@ -86,15 +105,15 @@ void GlslResourceBindingContext::emitResourceBindings(GenContext& context, const
     generator.emitLineBreak(stage);
 }
 
-void GlslResourceBindingContext::emitStructuredResourceBindings(GenContext& context,
-    const VariableBlock& uniforms, ShaderStage& stage, const std::string& structInstanceName,
-    const std::string& arraySuffix)
+void GlslResourceBindingContext::emitStructuredResourceBindings(GenContext& context, const VariableBlock& uniforms, 
+                                                                ShaderStage& stage, const std::string& structInstanceName,
+                                                                const std::string& arraySuffix)
 {
     ShaderGenerator& generator = context.getShaderGenerator();
     const Syntax& syntax = generator.getSyntax();
 
-    // Glsl struct need to be aligned. We make a best effort to base align struct members and add
-    // padding if required
+    // Glsl structures need to be aligned. We make a best effort to base align struct members and add
+    // padding if require.
```

Comment:
It looks like the original grammar here is correct, but this change makes it incorrect?  :)